aries-site/src/pages/templates/persistent-navigation.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+import { BulletedList, CardGrid, Meta, SubsectionText } from '../../components';
+import { MinimalSidebarExample } from '../../examples';
+import { ContentSection, Example, Layout, Subsection } from '../../layouts';
+import { getPageDetails, getRelatedContent } from '../../utils';
+
+const title = 'Persistent Navigation';
+const topic = 'Templates';
+const page = getPageDetails(title);
+const relatedContent = getRelatedContent(title);
+
+const Navigation = () => (
+  <Layout title={title}>
+    <Meta
+      title={title}
+      description={page.seoDescription}
+      canonicalUrl="https://design-system.hpe.design/templates/persistent-navigation"
+    />
+    <ContentSection>
+      <Subsection name={title} level={1} topic={topic}>
+        <SubsectionText>{page.description}</SubsectionText>
+        <Example
+        // code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/dashboards/DashboardExample.js"
+        // template
+        >
+          {}
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          Persistent navigation aids the user in orientation and provides
+          comfort in its presence, especially in information rich applications.
+          This pattern can be applied in both local and global contexts.
+        </SubsectionText>
+      </Subsection>
+      <Subsection
+        name="When to Use Persistent Navigation"
+        level={3}
+        gap="small"
+      >
+        <SubsectionText>
+          Persistent navigation should be used when:
+        </SubsectionText>
+        <BulletedList
+          items={[
+            `A user needs to easy access to move from one area of an 
+            application to another with minimal clicks.`,
+            `Elements such as branding or search require a ubiquitous 
```

Comment:
We should align the grammar here to be parallel to the first bullet point, maybe:
Branding or search functionality is present since these elements require a ubiquitous presence.